components/gantry/singleaxis/singleaxis.go

```diff
@@ -453,7 +453,11 @@ func (g *singleAxis) testLimit(ctx context.Context, pin int) (float64, error) {
 		}
 
 		elapsed := start.Sub(start)
-		if elapsed > (time.Second * 15) {
+		timeout := 15.0
```

Comment:
[nit] constants at top of the file, and set to a default name to something like `homingTimeout` with a comment.